src/VscPreviewExtension/src/dfmHttpClient.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import axios from 'axios';
+import { AxiosResponse } from 'axios'
+
+import { DfmServiceResult } from './dfmServiceResult';
+
+export class DfmHttpClient {
+    // TODO: make the urlPrefix configurable
+    private urlPrefix = "http://localhost:4001";
+
+    async SendPostRequestAsync(command: String, content: String): Promise<DfmServiceResult> {
```

Comment:
Use camelCase for function names: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines